web/pdf_viewer.js

```diff
@@ -1992,43 +1992,63 @@ class PDFViewer {
 
   /**
    * Increase the current zoom level one, or more, times.
-   * @param {number} [steps] - Defaults to zooming once.
    * @param {Object|null} [options]
    */
-  increaseScale(steps = 1, options = null) {
+  increaseScale(options = null) {
```

Comment:
Sorry, but this is a *breaking change* in the viewer-API which is why I specifically made the suggestion that I did in https://github.com/mozilla/pdf.js/pull/15886#discussion_r1060549906.

Hence if you want to make these change here, then you'll need to add fallback code that handles the old signature as well and also tag the PR with `[api-minor]`. E.g. something like this:
```suggestion
  increaseScale(options = null) {
    if (
      (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC")) &&
      typeof options === "number"
    ) {
      console.error("The `increaseScale` method-signature was updated, please use an object instead.");
      options = { steps: options };
    }
```